api/views.py

```diff
@@ -53,6 +55,19 @@ def get(self, request, *args, **kwargs):
         if return_format not in self.RETURN_FORMATS:
             return jsonResponse({'error': f'return_format should be one of those formats: {self.RETURN_FORMATS}.'}, status=400)
         text_manager = TextRequestAdapter(self.oref, versions_params, fill_in_missing_segments, return_format)
-        data = text_manager.get_versions_for_query()
-        data = self._handle_warnings(data)
+
```

Comment:
There are a lot of empty lines in this file, let's try and clean it up a bit :-) 
(Remove lines 9, 59, 68-73. 63 and 66 are up to you)